src/arrayMethodSort.js

```diff
@@ -3,9 +3,43 @@
 /**
  * Implement method Sort
  */
+
+function compareAsString(a, b) {
+  // linter didn't allow me to use the variables a and b here
+  const aNum = String(a);
+  const bNum = String(b);
+
+  if (aNum > bNum) {
+    return 1;
+  }
+
+  if (aNum < bNum) {
+    return -1;
+  }
+
+  return 0;
+}
+
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function(compareFunction = compareAsString) {
+    let count;
```

Comment:
Naming this variable something like isSorting would make it role more clear

Reply:
I totally agree here. It would also make sense to keep a boolean value here ;)